server/routes/probationPractitionerReferrals/interventionProgressView.ts

```diff
@@ -0,0 +1,41 @@
+import { TagArgs, TableArgs } from '../../utils/govukFrontendTypes'
+
+import InterventionProgressPresenter from './interventionProgressPresenter'
+
+export default class InterventionProgressView {
+  constructor(private readonly presenter: InterventionProgressPresenter) {}
+
+  private sessionTableArgs(tagMacro: (args: TagArgs) => string): TableArgs {
+    return {
+      head: this.presenter.sessionTableHeaders.map((header: string) => {
+        return { text: header }
+      }),
+      rows: this.presenter.sessionTableRows.map(row => {
+        return [
+          { text: `Session ${row.sessionNumber}` },
+          { text: `${row.appointmentTime}` },
+          { text: tagMacro(row.tagArgs as TagArgs) },
+          { html: `${row.linkHtml}` },
+        ]
+      }),
+    }
+  }
+
+  private readonly backLinkArgs = {
+    text: 'Back',
+    href: '/probation-practitioner/dashboard',
```

Comment:
does this currently exist?

Reply:
Yeah, but at the moment it's just the start of the "Make a referral" journey